### DISCUSSION TITLE
Can Osu anti-cheat block a user if the user just crashed the game without errors?

### DISCUSSION THREAD

**Discussion by ghost** [ppy/osu#22380] [Q&A]
Today there was such a problem after playing on one map, in the middle of the map the game crashed without errors (it often happens because my PC is very weak, so I disable services to improve performance) and so I went to the site and saw that my account was blocked I think that this is possible, since the anti-cheat thinks that the game is somehow broken by a crash, please, if anyone has any information, please explain.
I don’t consider it necessary to apply logs, since in a couple of minutes I learned to turn them on before that, I didn’t turn them on, or rather I wasn’t on the version that I needed

**Comment by Walavouchey** [comment]
If you've been restricted, please contact the [account support team](https://osu.ppy.sh/wiki/en/People/The_Team/Account_support_team) at <accounts@ppy.sh> and explain your situation in detail.

**Reply by ghost** [reply]
i do it but i think it problem called my pc because it bad pc with low ram and pc do close app for economy ram and i gain ban for it i say supportes but wait 6 mounts ban..its so long and bad for me because maybe its problem with my pc or anti cheat bag and real i lost all

**Reply by ghost** [reply]
I’m just afraid that it might happen that it’s a bug or a mistake and a ban was given and it’s not a fact that they will unban it, I’m afraid of this, that’s why I’m raising such a question because everything can be true

**Reply by Walavouchey** [reply]
It's not possible to be "banned because the game crashed". People cannot help you with account-related issues here, as this is a development forum. Again, contact support.

If you want to report a crash, provide the [game logs](https://github.com/ppy/osu/discussions/15188) for osu!(lazer) after the crash.

**Reply by ghost** [reply]
the logs I will be uploading are new, as I mentioned above, I recently learned about the logs and the current logs are no longer needed.

**Comment by ghost** [comment]
So my game close without errors what logs need?
